### PR TITLE
Update to consume relative path for post deployment script environment variable

### PR DESCRIPTION
**Pull request by shrimpy**

### CODE REVIEW

**Comment by ahmelsayed** [comment]
Is this a breaking change?

**Comment by davidebbo** [comment]
@ahmelsayed it's the other way around: it fixes a recent breaking change :)

**Comment by ahmelsayed** [comment]
I see

File: Kudu.Core/Deployment/Generator/ExternalCommandBuilder.cs [hunk 1]
```diff
@@ -129,7 +128,7 @@ public IList<string> GetPostBuildActionScripts()
             var defaultPath = Path.Combine(Environment.DeploymentToolsPath, PostDeploymentActions);
             if (!string.IsNullOrEmpty(System.Environment.GetEnvironmentVariable(SettingsKeys.PostDeploymentActionsDirectory)))
```
Comment (davidebbo) [inline Kudu.Core/Deployment/Generator/ExternalCommandBuilder.cs]: Use `DeploymentSettings.GetValue()` instead or it won't work with `.deployment`.

Comment (davidebbo) [inline Kudu.Core/Deployment/Generator/ExternalCommandBuilder.cs]: Also, store it in variable to avoid double fetching.

Comment (shrimpy) [inline Kudu.Core/Deployment/Generator/ExternalCommandBuilder.cs]: yep, will do. just confirmed from our old code (reset and deploy), we do support settings from .deployment file, but we only expect absolute file path.

i will update the logic to honor settings from both environment variable and deployment setting, relative path and absolute path.

Comment (davidebbo) [inline Kudu.Core/Deployment/Generator/ExternalCommandBuilder.cs]: If old logic only supported absolute file path, what caused the regression in #1894?

**Comment by davidebbo** [comment]
:shipit:

**Pull request merged**